src/category_theory/preadditive/of_biproducts.lean

```diff
@@ -0,0 +1,93 @@
+/-
+Copyright (c) 2022 Markus Himmel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Markus Himmel
+-/
+import category_theory.limits.shapes.biproducts
+import group_theory.eckmann_hilton
+
+/-!
+# Constructing a semiadditive structure from binary biproducts
+
+We show that any category with zero morphisms and binary biproducts is enriched over the category
+of commutative monoids.
+
+-/
+
+noncomputable theory
+
+universes v u
+
+open category_theory
+open category_theory.limits
+open eckmann_hilton
```

Comment:
Maybe don't `open` this, so that it is easy to spot where this is actually used. It will cost a few extra chars, but not that many. And you'll win in clarity, I think.